op-supervisor/supervisor/service.go

```diff
@@ -152,10 +156,24 @@ func (su *SupervisorService) initRPCServer(cfg *config.Config) error {
 		Service:       &frontend.UpdatesFrontend{Supervisor: su.backend},
 		Authenticated: false,
 	})
+
 	su.rpcServer = server
 	return nil
 }
 
+func (su *SupervisorService) initDBSync(cfg *config.Config) error {
+	syncCfg := sync.Config{
+		DataDir: cfg.Datadir,
+		Logger:  su.log,
+	}
```

Comment:
no `Chains` is set here, meaning all calls would be rejected?

Reply:
I've updated the code to set the chains, however it requires an additional call to `cfg.DependencySetSource.LoadDependencySet` which isn't ideal but seems like the highest ROI way to do it right now. I looked into moving the sync server into the backend, where we already have the depSet loaded, but that doesn't feel right either; it feels tacked on.

Let me know what you think.

Reply:
I'm good with reading the dependency set redundantly as you describe. I agree this isn't an appropriate backend component 👍